### PR TITLE
Added Label for Search Block Dropdown Form element

### PR DESCRIPTION
**Pull request by pranayjoshi**
Solved issue #5400 . I added the label "Sort by Dropdown" for better readability. I am not 100% sure if I had to add dropdown value as the label. But I think this is a safer bet as of now. Although I am open to modify the value.

### CODE REVIEW

File: src/components/manage/Blocks/Search/components/SortOn.jsx
Comment (stevepiercy) [inline src/components/manage/Blocks/Search/components/SortOn.jsx:76]: This must be internationalized. See recent comment at https://github.com/plone/volto/issues/5400#issuecomment-1826609113

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Your PR should not include changes from other PRs. Instead you should merge `main` into your branch, then push. See [Update your pull request from your fork](https://6.docs.plone.org/contributing/first-time.html#update-your-pull-request-from-your-fork).

**Comment by stevepiercy** [comment]
I see what happened. In your other PR https://github.com/plone/volto/pull/5413 you should not have worked on `main`. This makes it impossible to keep your `main` branch in synch with Volto's `main` branch. Instead you should have created a branch from `main`, then worked on your new branch, as described in [Write code](https://6.docs.plone.org/contributing/first-time.html#write-code).

You should get your repository sorted out first. This may mean breaking the guidance of not closing and reopening PRs for the same issue, but I think it may be necessary.

**Comment by pranayjoshi** [comment]
Hi! @stevepiercy thanks for pointing it out. Made all the required changes.

**Comment by stevepiercy** [comment]
Thank you. Next step: https://github.com/plone/volto/pull/5442#discussion_r1405330493

**Comment by stevepiercy** [comment]
> @stevepiercy I already did that

I don't think you read the comment to which I referred, specifically about internationalization.

**Comment by pranayjoshi** [comment]
My bad! I missed that part. Thanks for pointing out, I made the changes.

File: src/components/manage/Blocks/Search/components/SortOn.jsx
Comment (stevepiercy) [inline src/components/manage/Blocks/Search/components/SortOn.jsx:76]: This is not the way to support internationalization. The message needs to be translatable to other languages. See https://6.docs.plone.org/volto/recipes/i18n.html.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: src/components/manage/Blocks/Search/components/SortOn.jsx
Comment (nileshgulia1) [inline src/components/manage/Blocks/Search/components/SortOn.jsx:77]: You should move this to messages var for better readability.

Comment (pranayjoshi) [reply src/components/manage/Blocks/Search/components/SortOn.jsx:77]: do you have a suggestion for the variable name?

Comment (stevepiercy) [reply src/components/manage/Blocks/Search/components/SortOn.jsx:77]: See three lines previous for an example to follow. Again, you must follow the excellent documentation for to internationalize messages at https://6.docs.plone.org/volto/recipes/i18n.html.

**Comment by pranayjoshi** [comment]
@stevepiercy the current date of i18n run is conflicting please look into this.

**Comment by stevepiercy** [comment]
@pranayjoshi not my job, that's yours to resolve. The GitHub UI provides two ways to resolve the conflict.

Screenshot 2023-11-28 at 12 52 13 AM

**Comment by pranayjoshi** [comment]
> @pranayjoshi not my job, that's yours to resolve. The GitHub UI provides two ways to resolve the conflict.
> 
> Screenshot 2023-11-28 at 12 52 13 AM

It shows me an error saying "Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests."

**Comment by stevepiercy** [comment]
Did you try the command line?

**Comment by pranayjoshi** [comment]
> Did you try the command line?

I changed the edit date and I suppose this worked. Lemme know if it needs any changes

File: news/5400.bugfix
Comment (stevepiercy) [inline news/5400.bugfix:1]: By virtue of this filename, the change log will automatically include the issue number when it is released.

```suggestion
Improved accessibility by adding a label for the search block dropdown form element. @pranayjoshi
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor suggestion on the news item. Then this needs review by a maintainer.

File: news/5400.bugfix
Comment (pranayjoshi) [reply news/5400.bugfix:1]: Thanks. Would mind adding the review badge to my other PR.

**Review by stevepiercy — APPROVED** [review]

**Comment by ichim-david** [comment]
@pranayjoshi if you merge the conflicts again I think we can merge it this time

**Comment by pranayjoshi** [comment]
Screenshot 2023-12-20 at 2 55 53 AM
I am not allowed to merge these files. These requires write access

**Comment by ichim-david** [comment]
> Screenshot 2023-12-20 at 2 55 53 AM I am not allowed to merge these files. These requires write access

I was referring about merging the latest changes from main to fix the conflicts within your fork where you have permissions.
The pull request is merged only by someone with permission

**Comment by pranayjoshi** [comment]
> > Screenshot 2023-12-20 at 2 55 53 AM I am not allowed to merge these files. These requires write access
> 
> I was referring about merging the latest changes from main to fix the conflicts within your fork where you have permissions. The pull request is merged only by someone with permission

Thanks for pointing out! done with the changes

**Comment by davisagli** [comment]
This doesn't appear to actually solve the issue. The `label` prop is passed to the Select component, but it does not support a label prop (see https://react.semantic-ui.com/modules/dropdown/) and does not render anything based on it.

**Pull request closed without merging**
